src/main/java/org/sonar/plugins/stash/issue/MarkdownPrinter.java

```diff
@@ -120,12 +130,50 @@ private String formatTableList(String caption, List<String> items) {
     sb.append("|");
     sb.append(NEW_LINE);
     for (String item : items) {
-      sb.append("| ").append(item).append(" |").append(NEW_LINE);
+        sb.append("| ").append(item).append(" |").append(NEW_LINE);
+        if (this.includeFilesInOverview > 0 && issueToFilesMap.containsKey(item)) {
+            String files = String.join(", ", issueToFilesMap.get(item));
+            sb.append("| :arrow_right_hook: *Files: ").append(files).append("* |").append(NEW_LINE);

```

Comment:
Not sure about the emoticon. Could we use something more neutral, like a tab?

Reply:
Sadly spaces or tabs are not rendered so it was not possible to visually separate the files using them.

Now it will look something like this: ![screen shot 2018-05-07 at 15 44 03](https://user-images.githubusercontent.com/2048953/39705037-80acd20c-520d-11e8-9320-1f7f4342a0b3.png)

Reply:
Seems html works so I can use `&nbsp;&nbsp;&nbsp;&nbsp;` to separate them

Reply:
Please do. (I actually feel bad for advocating the use of `&nbsp` :-) )

Reply:
👍 :-D